### PR TITLE
rename cfgAce_Triggers to ACE_Triggers

### PR DESCRIPTION
**Pull request by commy2**

### CODE REVIEW

**Comment by esteldunedain** [comment]
While you're looking at explosives; the events on postInit could be replaced by "playerInventoryChanged" and "playerChanged". The "killed" EH also needs a bit more love to make it unit swithing compatible.

**Comment by commy2** [comment]
While we're at it, explosives should be attachable the same way chemlight are now.

File: addons/explosives/config.cpp [hunk 1]
```diff
@@ -18,7 +18,7 @@ class CfgPatches {
 #include "CfgMagazines.hpp"
 #include "CfgWeapons.hpp"
 #include "CfgVehicles.hpp"
-#include "CfgACE_Triggers.hpp"
+#include "CfgTriggers.hpp"
```
Comment (CorruptedHeart) [inline addons/explosives/config.cpp]: Shouldn't this be `ACE_Triggers.hpp` or `ACETriggers.hpp`?

Comment (esteldunedain) [inline addons/explosives/config.cpp]: Nod

**Pull request merged**